tests/azure/subnet_test.go

```diff
@@ -0,0 +1,48 @@
+package azure_test
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/NeowayLabs/klb/tests/lib/azure"
+	"github.com/NeowayLabs/klb/tests/lib/azure/fixture"
+)
+
+func genSubnetName() string {
+	return fmt.Sprintf("klb-subnet-tests-%d", rand.Intn(1000))
+}
+
+func testSubnetCreate(t *testing.T, f fixture.F) {
+	nsg := genNsgName()
+	f.Shell.Run(
```

Comment:
Would it not be easier to create a single script that creates all that you need ? This way you avoid calling Shell multiple times. The logs would probably make it clear where if failed if something goes wrong.